### ISSUE TITLE
awk: newline in string ... at source line 1 when running any fisher command

### ISSUE THREAD

**Issue by offbyone** [jorgebucaran/fisher#569]
### What's happening

Running any fisher command has been coming up with this:
```
fisher ls
awk: newline in string oh-my-fish/theme-agn... at source line 1
awk: newline in string oh-my-fish/theme-agn... at source line 1
awk: newline in string oh-my-fish/theme-agn... at source line 1
```

### System Configuration
fish shell version: 3.1.2
fisher version: 3.2.11
OS: 
```
ProductName:	Mac OS X
ProductVersion:	10.15.4
BuildVersion:	19E287
```
fishfile contents:
```
oh-my-fish/theme-agnoster
offbyone/autoenvstack
fisherman/pyenv
oh-my-fish/plugin-rustup
offbyone/fish-exa
```

### Debug
I turned on fish_trace and got this:

```
+ fisher ls
+++ set -q XDG_CACHE_HOME
+++ set XDG_CACHE_HOME /Users/offby1/.cache
+++ set -q XDG_CONFIG_HOME
+++ set XDG_CONFIG_HOME /Users/offby1/.config
+++ set -g fish_config /Users/offby1/.config/fish
+++ set -g fisher_cache /Users/offby1/.cache/fisher
+++ set -g fisher_config /Users/offby1/.config/fisher
+++ set -q fisher_path
+++ set -g fishfile /Users/offby1/.config/fish/fishfile
+++ for /Users/offby1/.config/fish/functions /Users/offby1/.config/fish/functions /Users/offby1/.config/fish/completions /Users/offby1/.config/fish/completions /Users/offby1/.config/fish/conf.d /Users/offby1/.config/fish/conf.d /Users/offby1/.cache/fisher
++++ if
++++ test ! -d /Users/offby1/.config/fish/functions
++++ end if
++++ if
++++ test ! -d /Users/offby1/.config/fish/functions
++++ end if
++++ if
++++ test ! -d /Users/offby1/.config/fish/completions
++++ end if
++++ if
++++ test ! -d /Users/offby1/.config/fish/completions
++++ end if
++++ if
++++ test ! -d /Users/offby1/.config/fish/conf.d
++++ end if
++++ if
++++ test ! -d /Users/offby1/.config/fish/conf.d
++++ end if
++++ if
++++ test ! -d /Users/offby1/.cache/fisher
++++ end if
+++ end for
+++ if
+++ test ! -e /Users/offby1/.config/fish/completions/fisher.fish
+++ end if
+++ if
+++ test -e /Users/offby1/.config/fish/conf.d/fisher.fish
+++ else
+++ end if
+++ if
+++ test -e /Users/offby1/.config/fish/fishfile
+++ test ! -e /Users/offby1/.config/fish/fishfile
+++ end if
++++ set -e 'argv[1]'
++++ if
++++ test -s /Users/offby1/.config/fish/fishfile
++++++ _fisher_fmt
++++++++ sed s\|\^\[\[:space:\]\]\*\|\|\;s\|\^/Users/offby1/.config/fisher/\|\|\;s\|\^\~\|/Users/offby1\|\;s\|\^\\.\\/\*\|/Users/offby1/\|\;s\|\^https\*:/\*\|\|\;s\|\^github\\.com/\|\|\;s\|/\*\$\|\|
++++++ sed 's|@.*||'
++++++ _fisher_parse -R
++++++++ set -e 'argv[1..2]'
++++++++ awk -v 'FS=[[:space:]]*#+' -v MODE=-R -v CMD= -v ARGSTR= \n\ \ \ \ \ \ \ \ BEGIN\ \{\n\ \ \ \ \ \ \ \ \ \ \ \ for\ \(n\ =\ split\(ARGSTR,\ a,\ \"\ \"\)\;\ i++\ \<\ n\;\)\ pkgs\[getkey\(a\[i\]\)\]\ =\ a\[i\]\n\ \ \ \ \ \ \ \ \}\n\ \ \ \ \ \ \ \ !NF\ \{\ next\ \}\ \{\ k\ =\ getkey\(\$1\)\ \}\n\ \ \ \ \ \ \ \ MODE\ ==\ \"-R\"\ \&\&\ !\(k\ in\ pkgs\)\ \&\&\ \(\$0\ =\ \$1\)\n\ \ \ \ \ \ \ \ MODE\ ==\ \"-W\"\ \&\&\ \(/\^\#/\ \|\|\ k\ in\ pkgs\ \|\|\ CMD\ !=\ \"rm\"\)\ \{\ print\ pkgs\[k\]\ \(sub\(\$1,\ \"\"\)\ \?\ \$0\ :\ \"\"\)\ \}\n\ \ \ \ \ \ \ \ MODE\ ==\ \"-W\"\ \|\|\ CMD\ ==\ \"rm\"\ \{\ delete\ pkgs\[k\]\ \}\n\ \ \ \ \ \ \ \ END\ \{\n\ \ \ \ \ \ \ \ \ \ \ \ for\ \(k\ in\ pkgs\)\ \{\n\ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ if\ \(CMD\ !=\ \"rm\"\ \|\|\ MODE\ ==\ \"-W\"\)\ print\ pkgs\[k\]\n\ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ else\ print\ \"fisher:\ cannot\ remove\ \\\"\"k\"\\\"\ --\ package\ is\ not\ in\ fishfile\"\ \>\ \"/dev/stderr\"\n\ \ \ \ \ \ \ \ \ \ \ \ \}\n\ \ \ \ \ \ \ \ \}\n\ \ \ \ \ \ \ \ function\ getkey\(s,\ \ a\)\ \{\n\ \ \ \ \ \ \ \ \ \ \ \ return\ \(split\(s,\ a,\ /@+\|:/\)\ \>\ 2\)\ \?\ a\[2\]\"/\"a\[1\]\"/\"a\[3\]\ :\ a\[1\]\n\ \ \ \ \ \ \ \ \}\n\ \ \ \
+++++ set -l file oh-my-fish/theme-agnoster\noffbyone/autoenvstack\nfisherman/pyenv\noh-my-fish/plugin-rustup\noffbyone/fish-exa
+++++ _fisher_ls
+++++++ for
+++++++ end for
+++++ awk -v FILE=oh-my-fish/theme-agnoster\noffbyone/autoenvstack\nfisherman/pyenv\noh-my-fish/plugin-rustup\noffbyone/fish-exa \n\ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ BEGIN\ \{\ for\ \(n\ =\ split\(FILE,\ f\)\;\ ++i\ \<=\ n\;\)\ file\[f\[i\]\]\ \}\ \$0\ in\ file\ \&\&\ //\n\ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \
+++++ sed 's|^/Users/offby1|~|'
+++++ _fisher_fmt
+++++++ sed s\|\^\[\[:space:\]\]\*\|\|\;s\|\^/Users/offby1/.config/fisher/\|\|\;s\|\^\~\|/Users/offby1\|\;s\|\^\\.\\/\*\|/Users/offby1/\|\;s\|\^https\*:/\*\|\|\;s\|\^github\\.com/\|\|\;s\|/\*\$\|\|
awk: newline in string oh-my-fish/theme-agn... at source line 1
awk: newline in string oh-my-fish/theme-agn... at source line 1
awk: newline in string oh-my-fish/theme-agn... at source line 1
++++ end if
```

**Comment by offbyone** [comment]
What's really nasty is that if I run `fisher` it removes the whole contents of the `fishfile` leaving it empty as well.

**Comment by jorgebucaran** [comment]
@offbyone I couldn't reproduce this by installing the packages you listed, but try disabling them one by one to rule out it's not any of them causing trouble here. 

It could also be something in your `config.fish`, so please share it as well.

**Comment by offbyone** [comment]
It's ... kind of a mystery, actually. I tried it tonight and it mostly worked. `fisher ls` came up silent and then installing packages worked.

The one thing I changed that I thought might be irrelevant at the time was that I deleted some of the old fzf bindings I'd installed with fisher some time ago, that had left some dead symlinks in `~/.config/fish/functions/__fzf*`... I wonder if those might have affected it.

(I am truly at a loss, though; I literally opened up the same running terminal as before, ran the command, and it was fine this time)
